staging/vhost-device-can/src/vhu_can.rs

```diff
@@ -1731,7 +1831,6 @@ mod tests {
     }
 
     #[test]
-    /// Test
     fn test_virtio_can_check_rx_eff_frame() {
         let controller =
             CanController::new("can0".to_string()).expect("Could not build controller");
```

Comment:
Does this test require to have `can0` in the host? My can device in the host is `vcan0` and the test still pass.